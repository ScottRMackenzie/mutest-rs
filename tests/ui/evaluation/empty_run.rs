//@ print-targets
//@ print-mutants
//@ run
//@ stdout
//@ stderr: empty
//@ run-flags: --exhaustive

fn non_mutable_fn() {}

#[test]
fn test() {
    assert_eq!((), non_mutable_fn());
}
