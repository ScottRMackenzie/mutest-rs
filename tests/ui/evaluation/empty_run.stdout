
@@@ targets @@@

tests -(0)-> non_mutable_fn at tests/ui/evaluation/empty_run.rs:8:1: 8:20 (#0)
  (0) test

targets: 1 total; 1 safe; 0 unsafe (0 tainted)

@@@ mutants @@@

0 mutants; 0 mutations; 0 safe; 0 unsafe (0 tainted); 0 batched; 0 unbatched

---

profiling reference test run

mutations: none. 0 detected (0 timed out; 0 crashed); 0 undetected; 0 total
     safe: none. 0 detected (0 timed out; 0 crashed); 0 undetected; 0 total
   unsafe: none. 0 detected (0 timed out; 0 crashed); 0 undetected; 0 total
